examples/matRad_example14_phantomBuilder.m

```diff
@@ -0,0 +1,169 @@
+%% Example: Generate your own phantom geometry
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2023 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% In this example we will show
+% (i) how to create arbitrary ct data (resolution, ct numbers)
+% (ii) how to create a cst structure containing the volume of interests of the phantom
+% (iii) generate a treatment plan for this phantom
+
+%% set matRad runtime configuration
+clear all;
+matRad_rc; %If this throws an error, run it from the parent directory first to set the paths
+
+%% Create a CT image series
+xDim = 200;
+yDim = 200;
+zDim = 100;
+
+ct.cubeDim      = [yDim xDim zDim]; % second cube dimension represents the x-coordinate
+ct.resolution.x = 2;
+ct.resolution.y = 2;
+ct.resolution.z = 3;
+ct.numOfCtScen  = 1;
+ 
+% create a ct image series with zeros - it will be filled later
+ct.cubeHU{1} = ones(ct.cubeDim) * -1000;
```

Comment:
These all could be arguments to the phantom-builder constructor, such that also the ct creation is managed there.